lsp-mode.el

```diff
@@ -6195,9 +6195,12 @@ information, for example if it doesn't support DocumentSymbols."
   "Render INPUT0, an `&DocumentSymbol', to a string.
 If SHOW-DETAIL? is set, make use of its `:detail?' field (often
 the signature)."
-  (let ((base (or (and show-detail? detail?) detail? name)))
-    (if deprecated? (propertize base 'face 'lsp-face-semhl-deprecated)
-      base)))
+  (let ((detail (and show-detail? (s-present? detail?)
+                     (propertize (concat " " (s-trim-left detail?))
+                                 'face 'lsp-lens-face)))
```

Comment:
lsp-lens-face is defined in lsp-lens.el which might not be present since lsp-lens.el is loaded only if lenses are loaded. I suggest creating a duplicate of that face, lets call it lsp-details-face in lsp-mode.el. Then, it makes sense to replace lsp-treemacs.el references to lsp-lens-face with lsp-details-face.

Reply:
We use a new `lsp-signature-face` instead now. This even makes it customizable.